coding/python_testing.rst

```diff
@@ -183,7 +183,7 @@ including :lclass:`~lsst.utils.tests.MemoryTestCase`:
 .. literalinclude:: unit_test_snippets/unittest_runner_example.py
    :linenos:
    :language: python
-   :emphasize-lines: 6, 16, 21
+   :emphasize-lines: 12, 17, 21
```

Comment:
Why the emphasize lines here? It's not clear why these particular lines are marked. You should either have some text calling out why those are marked, or not mark them.